k9mail/src/main/java/com/fsck/k9/view/MessageCryptoDisplayStatus.java

```diff
@@ -237,12 +252,15 @@ private static MessageCryptoDisplayStatus getDisplayStatusForPgpResult(CryptoRes
             }
         }
 
+        final long delta = signatureResult.getSignatureTimestamp().getTime() - messageSentDate.getTime();
```

Comment:
@Valodim @dschuermann Yes it is true, that this makes sense for XMPP too. But in XMPP the message frequency is much higher than in emails and also the frequency of short generic messages like "Yes". So the 48 hours makes sense for emails, but in XMPP I would actually set it to 10 seconds. .... just to make sure that no MITM replays the "Yes" that was given 10 seconds ago.

So even if we put this into OKC, it would be a per-application-setting. Right?

I thought that was the reason why you added the Signature-Timestamp-Information to the API and not the That-Signature-Seems-To-Be-Too-Old-Information ;)

or to approach this from another side: do you think it would be save to set this always to 10 seconds, no matter if this feature is user for email or chat? Because then it can clearly go into OKC and we could add an "SignatureIsTooOld" to the API.